src/main/java/io/kontur/disasterninja/DisasterNinjaApplication.java

```diff
@@ -1,5 +1,6 @@
 package io.kontur.disasterninja;
 
+import io.sentry.Sentry;
```

Comment:
Looks like an unused dependency

Reply:
It's instrumenting the whole app by evaluating this import

Reply:
Cool, ok:)